.github/workflows/Release.yml

```diff
@@ -0,0 +1,118 @@
+name: GitHub Release with APKs
+
+on:
+  push:
+    # tags:
+    # - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2    
+
+      - name: Copy CI gradle.properties
+        run: mkdir -p ~/.gradle ; cp .github/ci-gradle.properties ~/.gradle/gradle.properties
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Build all projects
+        run: ./scripts/gradlew_recursive.sh assembleDebug
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          #tag_name: "${{ github.ref }}"
+          #release_name: ${{ github.ref }}
+          draft: true # TODO update
+          release_name: "Test"
+          prerelease: false
```

Comment:
should this be true until stable? does it do anything apart from showing the label?

Reply:
I don't use pre-release personally. It doesn't do anything other than display a small note under the release name.